src/css/in-page.css

```diff
@@ -247,14 +350,23 @@
   padding: var(--spacingXs) var(--spacingSm);
   margin-bottom: var(--spacingSm);
   position: relative;
+  display: none;
+}
+
+.fx-relay-menu-masks-search-form.is-visible {
+  display: block;
 }
 
-.fx-relay-menu-masks-search-results.t-no-search-bar ul {
+.fx-relay-menu-masks-search .fx-relay-menu-masks-list.t-no-search-bar ul {
   max-height: none;
   overflow: visible;
 }
 
-.fx-relay-menu-masks-search-results ul {
+.fx-relay-mask-list-toggle-height .fx-relay-menu-masks-search .fx-relay-menu-masks-list ul {
+  max-height: 140px;
+}
+
+.fx-relay-menu-masks-search .fx-relay-menu-masks-list ul {
   max-height: 190px;
   overflow: scroll;
```

Comment:
Recommend using `overflow-y` instead so it only activates vertical scrolling.

Reply:
UI tweak nit: I also added `scrollbar-width: thin;` and set the footer to a fixed bottom sticky position to remove the odd spacing surrounding the mask list.

Before:
<img width="370" alt="image" src="https://user-images.githubusercontent.com/13066134/169396619-ddee417d-2ca9-4002-9e65-0fbce746716a.png">

After:

https://user-images.githubusercontent.com/13066134/169396746-abe2b846-bfba-4d5a-9a24-a35e5a417ce0.mov

Reply:
Whoa! TIL about `scrollbar-width: thin`. Nice fix! 

Added in ac4214cc685a936df860006fe1b376b448e81ccd